### PR TITLE
optimize: public ErrNoMultipartForm

### PR DESCRIPTION
**Pull request by li-jin-gou**
#### What type of PR is this?

optimize

#### Check the PR title.

- [x] This PR title match the format: `<type>(optional scope): <description>`.
- [x] The description of this PR title is user-oriented and clear enough for others to understand.
- [x] Attach the PR updating the user documentation if the current PR requires user awareness at the usage level. [User docs repo](https://github.com/cloudwego/cloudwego.github.io).

#### (Optional) Translate the PR title into Chinese.
- 公开 ErrNoMultipartForm

### CODE REVIEW

File: pkg/protocol/request.go [hunk 1]
```diff
@@ -65,8 +65,8 @@ import (
 )
 
 var (
-	errMissingFile     = errors.NewPublic("http: no such file")
-	errNoMultipartForm = errors.NewPublic("request has no multipart/form-data Content-Type")
+	ErrMissingFile     = errors.NewPublic("http: no such file")
```
Comment (welkeyever) [inline pkg/protocol/request.go:68]: 如果需要对外暴露，统一丢到pkg/common/errors下面吧

Comment (li-jin-gou) [reply pkg/protocol/request.go:68]: OK

File: pkg/protocol/request.go [hunk 2]
```diff
@@ -52,22 +52,20 @@ import (
 	"strings"
 	"sync"
 
+	"github.com/cloudwego/hertz/pkg/common/errors"
+
```
Comment (welkeyever) [inline pkg/protocol/request.go:56]: 空格

Comment (li-jin-gou) [reply pkg/protocol/request.go]: done

**Review by welkeyever — APPROVED** [review]

**Review by Duslia — APPROVED** [review]

**Review by wzekin — APPROVED** [review]

**Pull request merged**